### DISCUSSION TITLE
Using BOM as ingredients for recipie

### DISCUSSION THREAD

**Discussion by premudeshi** [alphabit-technology/erpnext-restaurant#22] [Q&A]
Hello.
We are trying to use ERPNext to manage inventory of our restaurant. I was wondering if there was a way to add a recipe builder (Similar to BOM for manufacturing) so that raw product inventory can be monitored.

**Comment by alphabit-technology** [comment]
But if you are preparing a BOM you can display it in the restaurant, or are you thinking of a model different from the existing one in ERP Next?

**Comment by premudeshi** [comment]
To use a BOM in ERPNext you have to create a work order for each item, which isn't ideal in a restaurant where there are 4-5 products on a single sales invoice on average. If there was some way of connecting raw materials to finished goods (like a recipe which would simply list the quantity required to make 1 unit of the item), it would allow us to track inventory of raw materials.

**Reply by alphabit-technology** [reply]
Oh perfect, I will study the case and review the process on Erpnext.

**Comment by alphabit-technology** [comment]
Can you describe an example of a product composed like this in the restaurant?

**Reply by premudeshi** [reply]
An example is: 1 sandwich, it contains raw items such as 3 tomatoes, 2 slices of cheese, 1 patty.  Let me try finding a workflow so it becomes easier to understand it.

**Reply by alphabit-technology** [reply]
Thanks

**Comment by alphabit-technology** [comment]
It is possible that I am confused, I know that BOMs are composite items based on Raw Material or other items, and these can be invoiced.

**Reply by premudeshi** [reply]
A BOM is just instructions for a finished product. It isn't a product on its own. To use a BOM in ERPNext, you would have to use the manufacturing module (by creating a work order followed by job card, here is the [documentation](https://docs.erpnext.com/docs/v13/user/manual/en/manufacturing/bill-of-materials) for this workflow), which would then deduct the stock of the raw items as well. It's a tedious process for a restaurant, so if there was a way to automate the entire workflow, it would help alot.

**Reply by alphabit-technology** [reply]
Excellent, I will read the documentation, it will be interesting to work on that feature.
